draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -602,7 +602,7 @@ signed JWT. See (#enveloping) for details.
 An SD-JWT is composed of the following:
 
 * the Issuer-signed JWT
-* The Disclosures
+* optionally the Disclosures
```

Comment:
this one might be controversial...

original feedback received was:

> According to clause 5.1 the Disclosures and the Key Binding JWT are both optional in an SD-JWT.
>
> Clause 5.11 explicitly specifies the serialized format of an SD-JWT if the Key Binding JWT is absent:
"In the case that there is no Key Binding JWT, the last
element MUST be an empty string and the last separating tilde character MUST NOT be omitted."
>
> Clause 5.11 does not explicitly specify the serialized format of an SD-JWT if the Disclosures are absent.
>
> Please specify explicitly the serialized format of an SD-JWT if the Disclosures are absent.
>
> Please indicate in the item list ("An SD-JWT is composed of the following…") that the Disclosures are optional (as already done for the Key Binding JWT).

Reply:
I'd interpreted "The Disclosures" to mean zero or more. So to me the optionality was implied.  But this seems fine too.

Reply:
```suggestion
* zero or more Disclosures
```

Reply:
I like the Dr.'s suggestion above of "zero or more Disclosures"